gamestonk_terminal/portfolio/portfolio_analysis/portfolio_view.py

```diff
@@ -16,13 +20,18 @@ def display_group_holdings(portfolio: pd.DataFrame, group_column: str):
     group_column : str
         Column to group by
     """
-
+    headers = [group_column, "value"]
     grouped_df = pd.DataFrame(portfolio.groupby(group_column).agg(sum)["value"])
+
+    if allocation:
+        total_value = grouped_df["value"].sum()
+        grouped_df["% allocation"] = grouped_df["value"] / total_value * 100
```

Comment:
Lets try to keep columns in the Dataframe as single strings - so "pct_allocation".  The header Is perfect

Reply:
Done!